docs/core/macos-prerequisites.md

```diff
@@ -44,6 +44,46 @@ Download and install the .NET Core SDK from [.NET Downloads](https://www.microso
 
 Download and install the .NET Core SDK from [.NET Downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult the [Known issues](https://github.com/dotnet/core/tree/master/release-notes/2.0) topic for the version you have installed.
 
+## Increase the maximum open file limit
+
+The default open file limit on macOS may not be sufficient for some .NET Core workloads, such as restoring projects or running unit tests.
+
+You can increase this limit by following these steps:
+
+Using a text editor, create a new file `/Library/LaunchDaemons/limit.maxfiles.plist`, and save the file with this content:
+
+```
```

Comment:
can you tag with the xml identifier like

```markdown
```xml
```

Reply:
OK, done.